src/web/app/src/components/SignUp/Forms/ChannelFeeds.tsx

```diff
@@ -8,6 +8,8 @@ const ChannelFeeds = () => {
     <RSSFeeds
       title="YouTube and Twitch"
       prompt="OPTIONAL: Enter your YouTube and/or Twitch channel"
+      promptExample1=""
+      promptExample2=""
```

Comment:
Here making the prop optional and assigning a default value like empty list , ensure that nothing will break.